conversation.go

```diff
@@ -109,6 +109,29 @@ func (api *Client) GetUsersInConversationContext(ctx context.Context, params *Ge
 	return response.Members, response.ResponseMetaData.NextCursor, nil
 }
 
+// GetAllConversationsForUser gets all conversations for a given user
+func (api *Client) GetAllConversationsForUser(
```

Comment:
not too keen on including these functions in the slack package. Only reason the user version exists is because it existed prior to implementing pagination for users.

I'd be happy to include them in the slackutilsx package though.